swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -95,171 +95,127 @@ Follow the instructions given in this section to develop the service.
-        string hospital;
-        string appointmentDate;
-    |};
-
-    type PaymentSettlement record {|
-        int appointmentNumber;
-        Doctor doctor;
-        Patient patient;
-        decimal fee;
-        boolean confirmed;
-        string card_number;
-    |};
-
-    type Payment record {|
-        int appointmentNo;
-        string doctorName;
-        string patient;
-        decimal actualFee;
-        int discount;
-        decimal discounted;
-        string paymentID;
-        string status;
-    |};
-
-    type ReservationRequest record {|
-        record {|
-            *Patient;
-            string cardNo;
-        |} patient;
-        string doctor;
-        string hospital_id;
-        string hospital;
-        string appointment_date;
-    |};
-
-    type ReservationResponse record {|
-        int appointmentNo;
-        string doctorName;
-        string patient;
-        decimal actualFee;
-        int discount;
-        decimal discounted;
-        string paymentID;
-        string status;
-    |};
-    ```
+    - Use `http:Created|http:NotFound|http:InternalServerError` as the return type to indicate that the response will be an `http:Created` response when the email is sent successfully to the user or the response will be an `http:NotFound` or `http:InternalServerError` response on error.
 
-   - The `ReservationRequest` record uses [record type inclusion](https://ballerina.io/learn/by-example/type-inclusion-for-records/) in the type of the `patient` field to include all the fields from the `Patient` record along with the `cardNo` field.
+        ![Define the resource](/learn/images/integration-tutorials/sending-emails-from-a-service/define_a_resource.gif)
 
-   - The initial record definitions can be generated using the "Paste JSON as record" VSCode command with the relevant JSON payloads and the records can then be modified as necessary.
+        The generated service will be as follows.
 
-6. Define the [HTTP service (REST API)](https://ballerina.io/learn/by-example/#rest-service) that has the resource function that accepts user requests, makes calls to the backend services to retrieve relevant details, and sends emails to the client with appointment details.
+        ```ballerina
+        service /healthcare on new http:Listener(8290) {
+            resource function post categories/[string category]/reserve(ReservationRequest payload) 
+                    returns http:Created|http:NotFound|http:InternalServerError {
+                
+            }
+        }
+        ```
+
+5. Define [configurable variables](https://ballerina.io/learn/by-example/#configurability) for the port on which the listener should listen and the URLs of the backend services. Also, define configurable variables for the host, username, and password of the SMTP client.
+
+    ![Define a configurable variable](/learn/images/integration-tutorials/sending-a-message-to-a-service/define_a_configurable_variable.gif)
```

Comment:
Still broken.

Reply:
The location of the GIF is wrong. (Please check the path)